docs/GettingStartedDocs/buildandsign.md

```diff
@@ -133,3 +132,49 @@ specify a `CONFFILE` that only changed that property:
 # Enclave settings:
 Debug=0
 ```
+
+## Signing an OP-TEE Enclave
+
+The `oesign` tool is currently SGX-only.  For OP-TEE enclaves, signing
+is instead performed by the `sign.py` script that comes with OP-TEE, and the
+script uses a key that is integrated into OP-TEE itself.  The signing key
+and script, among other artifacts, are exported to a "TA Dev Kit" during
+OP-TEE's build process. The Open Enclave SDK takes a CMake parameter at
+build time that specifies where to find the TA Dev Kit. This enables
+the `add_enclave` CMake function to locate sign.py and the TA signing key
+and apply them automatically to every TA built as part of the build process.
+
+There is currently no equivalent of a .conf file for OP-TEE, so all settings
+must be done using the `OE_SET_ENCLAVE_OPTEE` macro:
+
+```c
+#define TA_UUID                                            \
+    { /* 126830b9-eb9f-412a-89a7-bcc8a517c12e */           \
+        0x126830b9, 0xeb9f, 0x412a,                        \
+        {                                                  \
+            0x89, 0xa7, 0xbc, 0xc8, 0xa5, 0x17, 0xc1, 0x2e \
+        }                                                  \
+    }
+
+OE_SET_ENCLAVE_OPTEE(
+    TA_UUID,               /* UUID */
+    2 * 1024 * 1024,       /* Heap size, in bytes */
+    24 * 1024,             /* Stack size, in bytes */
```

Comment:
Perhaps these values could be made to match those in the example for SGX, for mere consistency to show how to express the same configuration using both macros:

```
OE_SET_ENCLAVE_OPTEE(
    TA_UUID,               /* UUID */
    4 * 1024 * 1024,       /* Heap size, in bytes */
    4 * 1024 * 1024,             /* Stack size, in bytes */
...
```

Alternately, the SGX ones could be made to reflect the OP-TEE ones seeing as OP-TEE has stricter memory boundaries in practice.

Reply:
Done